src/js/blocks/form-fields/inspect-field-options.js

```diff
@@ -300,10 +300,7 @@ export default class InspectorFieldOptions extends Component {
 
 		// Pass back to the block (and remove the ID so it isn't stored.
 		setAttributes( {
-			options: options.map( ( opt ) => ( {
-				...opt,
-				id: undefined,
-			} ) ),
+			options: options.map( ( { id, ...rest } ) => rest )
```

Comment:
@eri-trabiccolo check this out:

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#rest_in_object_destructuring

Reply:
This is very nice and fluffy and fills my heart of love.